spec/actors/hyrax/actors/file_actor_spec.rb

```diff
@@ -34,7 +34,7 @@ class FileSetWithExtras < FileSet
     end
     it 'uses the relation from the actor' do
       expect(CharacterizeJob).to receive(:perform_later).with(file_set, String, io.tempfile.path)
-      expect(actor.ingest_file(io)).to be_truthy
+      actor.ingest_file(io)
```

Comment:
Do we expect this to at least not return `false` here, or do the tests on the preceding and succeeding lines suffice? (I suspect the latter, that if the `CharacterizeJob` is called successfully and the reloaded mime type is as we expect it, we no longer need to check the return value of `#ingest_file`.)